lib/logger.js

```diff
@@ -358,11 +364,16 @@ Logger.prototype._flush = function(callback) {
  *  Populate short-hand for each supported Log Level
  */
 configs.LOG_LEVELS.forEach(function(level) {
-    var l = level.toLowerCase();
-    Logger.prototype[l] = function(statement, opts) {
+    var levelFuncationName = level.toLowerCase();
```

Comment:
Make this a `const` and fix the spelling

Reply:
We should set `"no-var": "error"` to enforce it: https://github.com/logdna/nodejs/blob/master/.eslintrc#L219

Reply:
Next PR for it